### PR TITLE
auto-push triage static files

### PR DESCRIPTION
**Pull request by BenTheElder**
xref: https://github.com/kubernetes/test-infra/issues/18726

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml [hunk 1]
```diff
@@ -598,6 +598,25 @@ postsubmits:
         resources:
           requests:
             memory: "1Gi"
+  - name: post-test-infra-upload-triage
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^triage/Makefile$|^triage/.+(\.html|\.js)$'
```
Comment (michaelkolber) [inline config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:605]: This misses CSS files. Also `.+` can include subdirectories, it's probably better to use `[^/]`

File: triage/Makefile [hunk 2]
```diff
@@ -15,12 +15,17 @@
 push:
   bazel run //images/builder -- --project=k8s-testimages --scratch-bucket=gs://k8s-testimages-scratch triage
 
-push_static:
-	gsutil -h 'Cache-Control: no-store, must-revalidate' -m cp -Z -a public-read index.html interactive.js model.js render.js style.css gs://k8s-gubernator/triage/
+STATIC_FILES=index.html interactive.js model.js render.js style.css
 
-push_staging:
-	gsutil -h 'Cache-Control: no-store, must-revalidate' -m cp -Z -a public-read index.html interactive.js model.js render.js style.css gs://k8s-gubernator/triage/staging/
+push-static:
```
Comment (michaelkolber) [inline triage/Makefile:20]: This is very similar to `push_static` above, might be confusing

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:605]: definitely want to catch css. shrug on subdirs, this just causes the job to run, it's cheap.

File: triage/Makefile (hunk 2 above)
Comment (BenTheElder) [reply triage/Makefile:20]: I think you're mis-reading the diff. `push_static` was renamed

Comment (michaelkolber) [reply triage/Makefile:20]: Ah yeah you're right

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml [hunk 3]
```diff
@@ -598,6 +598,25 @@ postsubmits:
...
           requests:
             memory: "1Gi"
+  - name: post-test-infra-upload-triage
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^triage/Makefile$|^triage/[^/]+(\.html|\.js|.css)$'
+    decorate: true
+    spec:
+      serviceAccountName: pusher
```
Comment (BenTheElder) [inline config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:608]: granted permission for k8s-gubernator bucket to pusher@k8s-prow.iam.gserviceaccount.com

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml [hunk 4]
```diff
@@ -598,6 +598,25 @@ postsubmits:
...
+        command:
+        - make
+        args:
+        - push-static
+        - push-staging
+        - push-go
+        resources:
+          requests:
+            memory: "1Gi"
     annotations:
```
Comment (michaelkolber) [inline config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:620]: I think you stole this from `post-test-infra-upload-oncall` by mistake, looks like they both need an `annotations`

Comment (BenTheElder) [reply config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:620]: heh I just pushed that fix

Comment (michaelkolber) [reply config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:620]: `post-test-infra-upload-oncall` still doesn't have its `annotations` though :)

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml [hunk 5]
```diff
@@ -604,6 +604,31 @@ postsubmits:
       testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
       testgrid-num-failures-to-alert: '1'
       description: Updates the html contents for go.k8s.io/oncall.
+  - name: post-test-infra-upload-triage
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^triage/Makefile$|^triage/[^/]+(\.html|\.js|.css)$'
```
Comment (michaelkolber) [inline config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:611]: For follow-up, should technically be `\.css`

**Pull request merged**